sentry-android-core/src/main/java/io/sentry/android/core/AndroidLogger.java

```diff
@@ -3,12 +3,22 @@
 import android.util.Log;
 import io.sentry.ILogger;
 import io.sentry.SentryLevel;
+import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-final class AndroidLogger implements ILogger {
+@ApiStatus.Internal
+public final class AndroidLogger implements ILogger {
 
-  private static final String tag = "Sentry";
+  private final String tag;
```

Comment:
```suggestion
  private final @NotNull String tag;
```

M: double check all the missing nullability annotations